alws/models.py

```diff
@@ -823,6 +823,33 @@ def __repr__(self):
     ),
 )
 
+
+TestRepositoryRoleMapping = sqlalchemy.Table(
```

Comment:
Do we really need this?

Reply:
I think so. Its used here:
`class TestRepository(PermissionsMixin, TeamMixin, Base):
    __tablename__ = "test_repositories"
    id: Mapped[int] = mapped_column(sqlalchemy.Integer, primary_key=True)
    name: Mapped[str] = mapped_column(
        sqlalchemy.Text, nullable=False, unique=True
    )
    url: Mapped[str] = mapped_column(
        sqlalchemy.Text, nullable=False, unique=True
    )
    tests_dir: Mapped[str] = mapped_column(sqlalchemy.Text, nullable=False)
    tests_prefix: Mapped[Optional[str]] = mapped_column(
        sqlalchemy.Text, nullable=True
    )
    packages: Mapped[List["PackageTestRepository"]] = relationship(
        "PackageTestRepository",
        back_populates="test_repository",
        cascade="all, delete",
    )
    team: Mapped["Team"] = relationship("Team", back_populates="test_repositories")
    roles: Mapped[List["UserRole"]] = relationship(
        "UserRole", secondary=TestRepositoryRoleMapping
    )`